gateway/auth_manager.go

```diff
@@ -63,15 +63,15 @@ func (b *DefaultSessionManager) ResetQuota(keyName string, session *user.Session
 	}).Info("Reset quota for key.")
 
 	rateLimiterSentinelKey := RateLimitKeyPrefix + keyName + ".BLOCKED"
+
 	// Clear the rate limiter
-	go b.store.DeleteRawKey(rateLimiterSentinelKey)
+	b.store.DeleteRawKey(rateLimiterSentinelKey)
 	// Fix the raw key
-	go b.store.DeleteRawKey(rawKey)
-	//go b.store.SetKey(rawKey, "0", session.QuotaRenewalRate)
+	b.store.DeleteRawKey(rawKey)
 
 	for _, acl := range session.AccessRights {
 		rawKey = QuotaKeyPrefix + acl.AllowanceScope + "-" + keyName
-		go b.store.DeleteRawKey(rawKey)
+		b.store.DeleteRawKey(rawKey)
 	}
```

Comment:
My first impression was to reject this change, since it adds 2+ more network calls, and slowdown this request.
But after I looked up where this function is used, and looks like it used only in rarely called key create/update API calls, e.g. we do not care there about performance. So should be good to keep it as it is.